Formula/rings.repl.rb

```diff
@@ -0,0 +1,16 @@
+class RingsRepl < Formula
+  desc "Rings: efficient Java/Scala library for polynomial rings"
+  homepage "http://ringsalgebra.io/"
+  url "https://raw.githubusercontent.com/PoslavskySV/rings/v.2.0.1/rings.repl/rings.repl", :using => :nounzip
+  sha256 "fe3fc145deb1750160c0a9b6098ea80a92ec0a841304b94da1af0e94143b1c00"
+
+  depends_on "ammonite-repl"
```

Comment:
This has a hard `Java 8` requirement currently. Compatibility with Java 9 is a requirement for new formulae. Therefore, this PR is blocked on https://github.com/lihaoyi/Ammonite/issues/675.

Also the :java requirement needs to be made explicit.